### PR TITLE
✨ [amp-render] support `binding="never"` as an alternative to `binding="no"`

### PR DESCRIPTION
**Pull request by dmanek**
Partial for #32477

### CODE REVIEW

**Comment by dmanek** [comment]
Hey @CrystalOnScript adding you for reviewing the updated documentation. FYI We recently made a decision to make `binding="refresh"` as the default and also support `binding="never"` as alternative to `binding="no"`.

File: extensions/amp-render/1.0/amp-render.js [hunk 1]
```diff
@@ -278,7 +279,8 @@ export class AmpRender extends BaseElement {
       this.mutateProps(
         dict({
           'render': (data) => {
-            if (this.element.getAttribute('binding') === 'no') {
+            const bindingValue = this.element.getAttribute('binding');
+            if (bindingValue === 'no' || bindingValue === 'never') {
```
Comment (caroqliu) [inline extensions/amp-render/1.0/amp-render.js:283]: Suggestion: Let's make all possible binding values into enums

File: extensions/amp-render/amp-render.md [hunk 2]
```diff
@@ -200,12 +200,17 @@ If we just want to display the German cars from the response, we can use the `ke
 
 ### `binding`
 
-Controls wheather to render on binding evaluations (e.g. `[text]`) in children on pages using `amp-render` and `amp-bind`.
+Controls whether to block render to evaluate bindings (e.g. `[text]`) in children on pages using `amp-render` and `amp-bind`.
```
Comment (caroqliu) [inline extensions/amp-render/amp-render.md:203]:
```suggestion
Controls whether to block render to evaluate bindings (e.g. `[text]`) in children on pages using `amp-render` and `amp-bind` together.
```

File: extensions/amp-render/amp-render.md [hunk 3]
```diff
@@ -200,12 +200,17 @@ If we just want to display the German cars from the response, we can use the `ke
...
--   `binding="refresh"`: Don't block render on initial load **(slower)**.
+-   `binding="never"` or `binding="no"`: Never block render **(fastest)**.
+-   `binding="refresh"` **(default)**: Don't block render on initial load **(faster)**.
+-   `binding="always"`: Always block render **(slow)**.
 
-If `binding` attribute is not provided, default is `refresh`.
+Essentially, `binding="always"` is `binding="refresh"` that also blocks to evaluate bindings on the initial load of `amp-render`.
+
+[tip type="important"]
+If `binding` attribute is not provided, default is `refresh`. This is departure from `amp-list` where the default is `always`. Due to this change, if specifying **only** a bound `src` attribute (`[src]`), it is imperative to include the `binding="always"` attribute on the component.
```
Comment (caroqliu) [inline extensions/amp-render/amp-render.md:212]:
```suggestion
If `binding` attribute is not provided, default is `refresh`. This is a departure from `amp-list` where the default is `always`. Due to this change, if specifying **only** a bound `src` attribute (`[src]`), it is imperative to include the `binding="always"` attribute on the component to be able to display templated content on the initial load.
```
Please also link to the amp-list `binding` documentation here for ease of reference.

**Review by caroqliu — APPROVED** [review]

**Pull request merged**